solidus_multi_domain.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.require_path = "lib"
   s.requirements << "none"
 
-  s.add_dependency "solidus", ['>= 1.1', '< 3']
+  s.add_dependency "solidus", ['>= 2.5', '< 3']
```

Comment:
This should probably be `>= 2.5.x` (so that we can include beta and rc releases)